module/map/map_fleet_preparation.py

```diff
@@ -68,32 +69,61 @@ def allow(self):
         return self.main.appear(self._choose)
 
     def clear(self):
-        self.main.device.click(self._clear)
-        self.main.device.sleep(self.FLEET_PREPARE_OPERATION_SLEEP)
+        while 1:
+            if not self.in_use():
+                break
+            self.main.device.click(self._clear)
+            # No need to sleep because the game reacts quickly here, and nothing
+            # goes wrong even if the clear button is clicked multiple times.
+            # If we sleep, the info bar of auto search will pop up, then we may
+            # have to waste a few seconds for it to disappear in in_use().
+            self.main.device.screenshot()

```

Comment:
```
2021-02-12 23:46:23.636 | INFO | [Fast_Forward] on
2021-02-12 23:46:23.651 | INFO | [Auto_Search] off
2021-02-12 23:46:23.652 | INFO | Click ( 943,  512) @ MAP_PREPARATION
2021-02-12 23:46:24.027 | INFO | Using fleet: [1, 2, 0]
2021-02-12 23:46:24.029 | INFO | Click (1121,  331) @ FLEET_2_CLEAR
2021-02-12 23:46:24.393 | INFO | Click (1132,  344) @ FLEET_2_CLEAR
2021-02-12 23:46:27.761 | INFO | Click (1037,  199) @ FLEET_1_CHOOSE
2021-02-12 23:46:28.448 | INFO | Current selected: [1]
2021-02-12 23:46:28.449 | INFO | Click (1048,  200) @ FLEET_1_CHOOSE
2021-02-12 23:46:29.070 | INFO | Click (1031,  332) @ FLEET_2_CHOOSE
2021-02-12 23:46:29.756 | INFO | Current selected: [1]
2021-02-12 23:46:29.756 | INFO | Click (1113,  443) @ FLEET_2_BAR_INDEX_2
2021-02-12 23:46:30.421 | INFO | Click (1072,  604) @ FLEET_PREPARATION
2021-02-12 23:46:35.948 | INFO | Enemy searching appeared.
2021-02-12 23:46:36.297 | INFO | [Fleet_Lock] off
2021-02-12 23:46:36.297 | INFO | <<< MAP INIT >>>
```
On my side, it double clicked FLEET_2_CLEAR, and still waiting for info bar to disappear.
I re-cropped `FLEET_*_IN_USE` to avoid detecting the info bar, also do the trick. 5113cf9d554872c891b8a0931afd2000067f3e6d